src/dbos.ts

```diff
@@ -336,11 +336,21 @@ export class DBOS {
     const logger = DBOS.logger;
     if (DBOS.runtimeConfig && DBOS.runtimeConfig.runAdminServer) {
       const adminApp = DBOSHttpServer.setupAdminApp(executor);
-      await DBOSHttpServer.checkPortAvailabilityIPv4Ipv6(DBOS.runtimeConfig.admin_port, logger as GlobalLogger);
-
-      DBOS.adminServer = adminApp.listen(DBOS.runtimeConfig.admin_port, () => {
-        DBOS.logger.info(`DBOS Admin Server is running at http://localhost:${DBOS.runtimeConfig?.admin_port}`);
-      });
+      try {
+        await DBOSHttpServer.checkPortAvailabilityIPv4Ipv6(DBOS.runtimeConfig.admin_port, logger as GlobalLogger);
```

Comment:
What does this do?

Reply:
Checks if the ports are busy or not on the machine -- and throws if they are